src/components/tooltip/Tooltip.tsx

```diff
@@ -275,7 +285,15 @@ class Tooltip extends React.Component<TooltipProps, State> {
             <TetherComponent ref={this.tetherRef} {...tetherProps}>
                 {React.cloneElement(React.Children.only(children) as React.ReactElement, componentProps)}
                 {showTooltip && (
-                    <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
+                    /* eslint-disable-next-line jsx-a11y/no-noninteractive-element-interactions */
```

Comment:
We should avoid disabling these rules, as they are present for good reason. Can we find another way?

Reply:
Hi, see
https://github.com/evcohen/eslint-plugin-jsx-a11y/blob/master/docs/rules/no-noninteractive-element-interactions.md
For the lint rule and recommended solution. I think the recommended solution is good to go except it modifies DOM structure so practically it might not be the choice.

Reply:
So it seems like there a few options to move forward. Can you investigate those and choose one as appropriate?

Reply:
Hi, Jeff,
yes, the choices I considered is 
1) write code to attach event handler to DOM that prevent propagation
2) attach onClick to current div
3) attach onClick to a new div created inside current div

The drawback for 3) is it creates a new div that means a lot more testing and potentially more unexpected work. The drawback for 1) is it is more complex is less canonical. It doesn't really create much of a benefit. End of day we are attaching an event listener to a DOM element and in this case we have to find the DOM element before we attach, so a lot of complexity for a benefit that I can't see much.

Reply:
changed to render a presentation div to capture the event if shouldStopEventPropagation is true